src/main/frontend/extensions/code.css

```diff
@@ -1,12 +1,16 @@
 .extensions__code {
   @apply relative;
   z-index: 0;
-
+  display: flex;
+  flex-direction: row-reverse;
+  flex-wrap: nowrap;
```

Comment:
This fix breaks code editing styles.

FYI: In my custom.css:
```css
.extensions__code .CodeMirror-scroll {
   margin-right: 0 !important;
}

```

Reply:
> This fix breaks code editing styles.
> 
> FYI: In my custom.css:
> 
> ```css
> .extensions__code .CodeMirror-scroll {
>    margin-right: 0 !important;
> }
> ```

thx,I will check it.

Reply:
> This fix breaks code editing styles.
> 
> FYI: In my custom.css:
> 
> ```css
> .extensions__code .CodeMirror-scroll {
>    margin-right: 0 !important;
> }
> ```

Sorry,I still can't pass the e2e-test.I hadn't use playwright/test before.I had read some docs,but that can't help me to pass the  test.[fail test case is here](https://github.com/logseq/logseq/blob/d7b1ce51e368a8ea7ae2020070144b7e0441abcf/e2e-tests/code-editing.spec.ts#L124).
It feels like an input and output value is returned incorrectly.
I tested manually,but can't repeat the failed test case.


**This is the gif**

**mobile-before**
![codemobilebefore](https://user-images.githubusercontent.com/46164858/145953090-733aeda3-d2c6-4720-91df-8b22461c9cb0.gif)
**mobile-after**
![codemobileafter](https://user-images.githubusercontent.com/46164858/145953107-4c9afdd4-24af-427f-99ab-c35422747795.gif)
**desktop-before**
![codebefore](https://user-images.githubusercontent.com/46164858/145953147-8ce42ec7-bdd4-494c-8886-31dc05e4a339.gif)
**desktop-after**
![codeafter](https://user-images.githubusercontent.com/46164858/145953157-0f65d8a7-348f-4574-bce5-2353e5e87216.gif)

Reply:
Could you try a shorter code snippet? (shorter than screen width)

Reply:
> Could you try a shorter code snippet? (shorter than screen width)

Yes,finally i pass the test.Thx for the  help.By the way,maybe i should set up development environment at first.